src/components/navbar/Navbar.jsx

```diff
@@ -0,0 +1,25 @@
+import { Link } from "react-router-dom";
+import "./Navbar.css";
+const Navbar = () => {
+  return (
```

Comment:
[OPTIONAL] There are a couple of errors in your applications when I build your application with `npm run build`. It might not be important right now, but you could run into issues when you try to deploy your application.

I highly recommend that you look into it.

in `src/components/navbar/Navbar.jsx`

![image](https://user-images.githubusercontent.com/38088818/206558204-e2a0871f-d710-43fd-8d22-212ffd50d026.png)

in `src/routes/calculator/CalculatorPage.jsx`

![image](https://user-images.githubusercontent.com/38088818/206558428-63d8ac60-76e5-45ec-94c2-11a1b8052503.png)